templates/codepipeline-cf-template.yml

```diff
@@ -294,24 +343,25 @@ Resources:
   S3CodePipeline:
     Type: "AWS::S3::Bucket"
     Properties:
-      BucketName: !Join [ "", [!Ref CodePipelineBucketPrefix,  "-", !Ref 'AWS::StackName', !Ref 'Environment', "-", !Ref "AWS::Region"]]
+      BucketName: !Join [ "", [!Ref CodePipelineBucketPrefix,  "-", !Ref 'AWS::StackName', !Ref 'Environment']]
```

Comment:
Might want to add a dash between Stackname and Env (like zoomef-dev) instead of zoomefdev, thoughts?

Reply:
like we do on line 352 below

Reply:
Yes, sounds good. Pretty sure I removed that dash deliberately because CloudFormation was complaining about bucket name being too long/invalid.

Added it again, redeployed and did not complain this time.